docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -54,62 +72,76 @@ See our documentation [getting access to AWS](index.md#getting-access-to-aws) fo
     ```
 
-    The command `export AWS_PROFILE=openscpca` will define your AWS profile name as `openscpca` for the duration of your terminal session.
-    Defining this environment variable is helpful because the `download-data.py` script also needs your profile name to download data from S3.
+- You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
 
-    Note that you can also [add this profile definition to your shell profile file](../../technical-setup/environment-setup/configure-aws-cli.md#storing-your-aws-profile-name) so that it will always be defined in any terminal session.
+    !!! warning
+        **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
+        Please download the full data release with caution!**
 
-We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+    ```sh
+    ./download-data.py
+    ```
 
-You can list all the options for the download data script by running the following from the root directory of the repository:
+- If you prefer to work with `AnnData` files, use the `--format` option as follows:
 
-```sh
-./download-data.py --help
-```
+    ```sh
+    ./download-data.py --format AnnData
+    ```
 
-You can run the download script with all default options to download all processed samples from the most recent release in `SingleCellExperiment` format:
+- To download a specific release, you can use the `--release` flag as follows:
 
-!!! warning
-    **The full data release download is quite large: over 35 GB for `SingleCellExperiment` format, and over 100 GB for `AnnData` format.
-    Please download the full data release with caution!**
 
-```sh
-./download-data.py
-```
 
-If you prefer to work with `AnnData` files, use the `--format` option as follows:
+<!--
 
-```sh
-./download-data.py --format AnnData
-```
+TODO: This section needs to be reworked to avoid unexpected downloads. We should recommend --use-release instead.
 
-To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+These commands will download either test data or results, respectively, and will
 
 ```sh
-./download-data.py --dryrun
+./download-data.py --release current
+
+./download-results.py --release current
 ```
+If you had already downloaded the most recent data or results, this will not repeat downloading the files you already have.
 
-If you're only working with a subset of the data, you can use the `--projects` or `--samples` to download select samples (note: these options are mutually exclusive):
+-->
 
-```sh
-# use the --projects option
-./download-data.py --projects SCPCPXXXXXX
+- To review what samples would be downloaded without performing the download yet, you can use the `--dryrun` option as follows.
```

Comment:
I think it might make sense to put this option after the projects and maybe show it combined with other options as well. 
```suggestion
- To review the files would be downloaded without performing the download yet, you can use the `--dryrun` option as follows.
```